app/_locales/en/messages.json

```diff
@@ -1673,6 +1676,17 @@
   "networkStatus": {
     "message": "Network status"
   },
+  "networkStatusBaseFeeTooltip": {
+    "message": "The base fee is set by the network and changes every 8-12 seconds. Our $1 and $2 options account for sudden increases.",
+    "description": "$1 and $2 are bold text for Medium and Aggressive respectively."
+  },
+  "networkStatusPriorityFeeTooltip": {
+    "message": "Range of prority fees (aka “miner tip”). This goes to miners and incentivizes them to prioritize your transaction."
+  },
+  "networkStatusStabilityFeeTooltip": {
+    "message": "Gas fees are $1 relative to the past 24 hours.",
```

Comment:
We are using the past 20,000 blocks to determine the network status (source: https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/master/src/eip1559/getInfuraGasPrices.ts#L21 and https://gitlab.com/ConsenSys/codefi/products/metaswap/gas-api/-/blob/master/src/eip1559/constants.ts#L24). Using an average of 13-14 seconds per block this comes out to ~72-78 hours. What do we feel about changing this to "Gas fees are $1 relative to the past 72 hours"?

Reply:
Good suggestions, let me check with Jake about this.

Reply:
Hi @jakehaugen : can you plz suggest about above.

Reply:
Sounds good!

Reply:
I did the change 👍